lib/bolt/plugin.rb

```diff
@@ -36,6 +36,13 @@ def initialize(plugin_name, hook)
           super("Plugin #{plugin_name} does not support #{hook}", 'bolt/unsupported-hook')
         end
       end
+
+      class LoadingDisabled < PluginError
+        def initialize(plugin_name)
+          msg = "Cannot load plugin #{plugin_name}: plugin loading is disabled"
+          super(msg, 'bolt/plugin-loading-disabled', { plugin_name: plugin_name })
```

Comment:
hmm, are there any implications to using a keyword for the key here? I feel like everywhere else in bolt errors we use a string...

Reply:
That's just me not knowing Bolt conventions. Updated to use a string.